src/components/SignInForm.jsx

```diff
@@ -16,7 +16,14 @@ const SignIn = (props, routeProps) => {
 		setFieldType(fieldType === 'Password' ? 'text' : 'Password');
 	};
 
-
+	if (formIncomplete(props) === true || props.values.ContactID === null) {
+		props.history.push('/ServiceRequestForm');
+		props.setFieldValue("userNeedsToLoginError", "Please log in to continue");
```

Comment:
I think this string is stored in a json file. I'm not sure it should be there, but it shouldn't be duplicated. Let me know how you think we should handle.

Reply:
this was left over code from a previous version. Has been deleted on all pages.

Reply:
I'm still seeing it in both files.